### PR TITLE
86cuvkum8/bug-fixes-trip-card-accordion

### PR DESCRIPTION
**Pull request by seraGeorge**
### Changes

- Change format for bus type and seat type in accordion.
- Margin between seat layout and selected seats display when layout is vertical. Also add bottom margin.
- Book Now button and fare detail overflows in iphone se.
- Seat type and bus type overflowing in mobile view.
- Only one type is mentioned in bus type section of accordion where values have both bus and seat type.
- Wrap the text of departure and arrival in accordion in mobile view. Only dots are visible.

### CODE REVIEW

**Review by akshay-krishna — APPROVED** [review]

File: src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-            <Stack direction={'row'} spacing={1} className='title-container'>
+        <Stack direction={'row'} gap={'2rem'}>
+            <Stack direction={'row'} spacing={1} className="title-container">
                 {icon}
-                {title && (
-                    <Typography variant="body2" className="title">
-                        {title}:
-                    </Typography>
-                )}
+                {isMinWidth
```
Comment (sania-salim) [inline src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx]: Retain the title of field in small screen, user cannot distinguish between the icons. Dots are still visible in mobile screen. Please wrap the p tag of field in a div and provide flex-wrap.

File: src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
+                    ? title && (
+                          <Typography variant="body2" className="title">
+                              {title}:
+                          </Typography>
+                      )
+                    : ''}
             </Stack>
             {value && (
                 <Typography variant="body2" className="value">
```
Comment (sania-salim) [inline src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx]: Enum is being displayed. Change to text depending on the enum.

File: src/components/ui/TripCardAccordion/AccordionDetails/TripCardDetails.tsx [hunk 3]
```diff
@@ -121,7 +121,11 @@ export const TripCardDetails = ({
                     <TripCardDetailsLoader />
                 ) : (
                     <>
-                        <Stack direction={'column'} spacing={2}>
+                        <Stack
```
Comment (sania-salim) [inline src/components/ui/TripCardAccordion/AccordionDetails/TripCardDetails.tsx:125]: Change loader in trip card accordion for uniformity.

**Review by sania-salim — CHANGES_REQUESTED** [review]

File: src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx (hunk 1 above)
Comment (seraGeorge) [reply src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx]: Resolved in 2547706eb24bc5825727a17133e4bf3828bb320e

File: src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx (hunk 2 above)
Comment (seraGeorge) [reply src/components/ui/TripCardAccordion/AccordionDetails/components/DetailItem.tsx]: Resolved in c15fd5163ec359f4531e1db32885dbd5c2df7a9a

File: src/components/ui/TripCardAccordion/AccordionDetails/TripCardDetails.tsx (hunk 3 above)
Comment (seraGeorge) [reply src/components/ui/TripCardAccordion/AccordionDetails/TripCardDetails.tsx:125]: Resolved in 7e039dcf9fe8b009e9ec7adfc9fcffac7e90a8c6

**Review by sania-salim — APPROVED** [review]

**Pull request merged**